src/components/TestData/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+import TestFileView from '../TestFileView';
+import TestStatusBar from '../TestStatusBar';
+
+const TestData: React.FC = () => {
+  return (
+    <Wrapper>
+      <TestStatusBar />
+      <TestDetailsWrapper>
+        <Left>
+          <TestFileView />
+        </Left>
+        <Right>Right</Right>
+      </TestDetailsWrapper>
+    </Wrapper>
+  );
+};
+
+const Wrapper = styled.section`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  min-height: 1200px;
```

Comment:
Can see a lot of hard-coded values, should have fix these so that it can be responsive